packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.types.ts

```diff
@@ -239,6 +239,9 @@ export interface IGroupDividerProps {
   /** The indent level of the group. */
   groupLevel?: number;
 
+  /** The width unit used in the spacer for the group. */
+  spacerWidthUnit?: number;
```

Comment:
widthUnit makes me think this is px/em/% or something. If it's just a number, spacerWidth should bre fine.

Reply:
I wanted to convey that this is not the actual space that gets added, but a "unit" that gets multiplied based on the level. Should I just change it to `spacerWidth` and add this description in the docs?

Reply:
Yeah, if the multiplication logic is simple enough to explain, adding it to the description is fine.

Reply:
I updated the comment to say:
>The spacer width corresponding to a single level. This is multiplied by the groupLevel to get the full spacer width for the group.

I still think that `spacerWidth` can be misleading (not everybody reads the docs). Would something like `spacerUnitWidth` work instead?

Reply:
spacerLevelWidth? indentWidth? levelWidth? spacerUnitWidth is certainly better than spacerWidthUnit.